Caboodle/SecureStorage/SecureStorage.shared.cs

```diff
@@ -0,0 +1,36 @@
+﻿#if !NETSTANDARD1_0
+using System.Security.Cryptography;
+#endif
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class SecureStorage
+    {
+        // md5(Caboodle.SecureStorage)
+        const string localDirName = "d2f74ffc823f1d9f015ee8edf1463498";
```

Comment:
On most platforms we'll need to store data in files.  How do we feel about this?